digitalstrategie/assets/scss/_variables.scss

```diff
@@ -168,10 +168,10 @@ $transition-collapse-width:   width .35s ease !default;
 
 // scss-docs-start font-variables
 // stylelint-disable value-keyword-case
-$font-family-sans-serif:      "Arial", "Helvetica", sans-serif !default;
+$font-family-sans-serif:      "BerlinType", sans-serif !default;
 $font-family-monospace:       SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace !default;
 // stylelint-enable value-keyword-case
-$font-family-base:            var(--#{$variable-prefix}font-sans-serif) !default;
+$font-family-base:            var(--#{$variable-prefix}font-monospace) !default;
 $font-family-code:            var(--#{$variable-prefix}font-monospace) !default;
```

Comment:
um base shoudl stay as sans serif, we should just set sans serif to our prefered font, like it was